src/main/java/com/perennial/doctorappointmentbooking/helper/AppointmentHelper.java

```diff
@@ -71,10 +70,10 @@ public static boolean checkExcelFormatOfAppointment(MultipartFile file)
                                 appointment.setAppointmentTime(currentCell.getStringCellValue());
```

Comment:
Change the implementation of the class to separate the reading and parsing concerns.